spec/helpers/medication_refill_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe MedicationRefillHelper, type: :helper do
+  include CalendarHelper
+  include Devise::Test::ControllerHelpers
+
+  describe '#save_refill_to_google_calendar' do
+    let(:user) { FactoryBot.create(:user1) }
+    let(:medication) { FactoryBot.create(:medication, user_id: user.id) }
+    let(:exception_text) { 'RESCUE INVOKED' }
+
+    before do
+      allow_any_instance_of(helper.class).to receive(:return_to_sign_in).and_return(exception_text)
+      sign_in user
+    end
+
+    context 'when the user has the google oauth2 enabled and he need a new refill reminder needed' do
```

Comment:
Let's remove the "he" pronoun to be neutral. You can replace it with "they" instead.